DEV-9/DEV-9.Tests/MailReceivingTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using DEV_9.PageObjects.MailRu;
+using DEV_9.PageObjects.YandexMail;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace DEV_9.Tests
+{
+    /// <summary>
+    /// These tests check mailboxes for sending and receiving messages.
+    /// </summary>
+    [TestFixture]
+    public class MailReceivingTests
+    {
+        private IWebDriver YandexDriver;
+        private IWebDriver MailRuDriver;
+        private PageObjects.YandexMail.InboxPage YandexInboxPage;
+        private MailRuHomePage MailRuPage;
+        private string MailRuLogin = "babeyfakemail@mail.ru";
+        private string MailRuPassword = "MegaPassword";
+        private string Recipient = "BabeyFakeMail@yandex.ru";
+        private string MessageText = "Mega Message";
+        private string AnswerText = "Super nickname";
+
+        /// <summary>
+        /// Logging in to Mail.ru, sending messages to Yandex and logging into Yandex mail 
+        /// </summary>
+        [SetUp]
```

Comment:
SetUp is just a base functionality for tests. it is not needed to send some onformation in it